discovery-provider/src/utils/elasticdsl.py

```diff
@@ -7,7 +7,7 @@
 es_url = os.getenv("audius_elasticsearch_url")
 esclient = None
 if es_url and not esclient:
-    esclient = Elasticsearch(es_url)
+    esclient = Elasticsearch(es_url, request_timeout=60)
```

Comment:
is this to timeout those potentially large requests like get feed for large followings?